internal/prefixcollector/exclude_prefix_pool.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prefixcollector
```

Comment:
Why do we need this file? Can we use exclude prefix pool from SDK?

Reply:
As I understand, prefix pool was designed to work with available (not excluded) prefixes. It allows us to exclude prefixes from current pool and release excluded back to available ones. I didn't find way of collecting all 
non-overlapping exclude prefixes using pool methods.